src/components/MatchingList.tsx

```diff
@@ -6,42 +6,66 @@ import { formatStopName } from '../utils';
 interface MatchingListProps {
     userFareStages: UserFareStages;
     stops: Stop[];
+    selectedFareStages: string[];
 }
 
-const MatchingList = ({ userFareStages, stops }: MatchingListProps): ReactElement => (
+const MatchingList = ({ userFareStages, stops, selectedFareStages }: MatchingListProps): ReactElement => (
     <div>
         <div className="matching-wrapper">
             <div className="govuk-heading-s matching-stop-header">Stop name</div>
             <div className="govuk-heading-s naptan-code-header">Naptan code</div>
             <div className="govuk-heading-s fare-stage-header">Fare stage</div>
         </div>
-        {stops.map((stop, index) => (
-            <fieldset key={stop.atcoCode} className="govuk-fieldset">
-                <div className="matching-wrapper">
-                    <label className="govuk-label matching-stop-name" htmlFor={`option-${index}`}>
-                        {formatStopName(stop)}
-                    </label>
-                    <label className="govuk-label naptan-code" htmlFor={`option-${index}`}>
-                        {stop.naptanCode}
-                    </label>
-                    <div className="farestage-select-wrapper">
-                        <select
-                            className="govuk-select farestage-select"
-                            id={`option-${index}`}
-                            name={`option${index}`}
-                        >
-                            <option value="">Not Applicable</option>
-
-                            {userFareStages.fareStages.map((stage: FareStage) => (
-                                <option key={stage.stageName} value={JSON.stringify({ stop, stage: stage.stageName })}>
-                                    {stage.stageName}
-                                </option>
-                            ))}
-                        </select>
+        {stops.map((stop, index) => {
```

Comment:
Struggling to understand what's going on here - would be good to have a chat to grok it :)